ui-v2/src/hooks/variables.ts

```diff
@@ -11,82 +11,91 @@ import {
 
 type UseVariablesOptions =
 	components["schemas"]["Body_read_variables_variables_filter_post"];
+type Variables = UseVariablesOptions["variables"];
```

Comment:
We can use the existing schema from the type generation to avoid coupling between the types.
```suggestion
type VariablesFilter = components["schemas"]["VariableFilter"];
```